src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Profile.vue

```diff
@@ -169,10 +169,10 @@ export default defineComponent({
   setup: async () => {
     const userProfile = await getProfile();
     const userStore = useUserStore();
-
+    const verifiedPreviousNamesFiltered = userStore.verifiedPreviousNames.filter((name) => name.source !== "NameLog");
     userStore.setUserProfile(userProfile);
 
-    return { userProfile, userStore };
+    return { userProfile, userStore, verifiedPreviousNamesFiltered };
```

Comment:
I was wondering if we should just filter previous names in the store. Since we don't want any NameLogs for previous names on the frontend.

Reply:
done, adjusted EceEducation component as well